.eslintrc.js

```diff
@@ -1,47 +1,48 @@
 module.exports = {
-  "globals": {
-    "gettext": true,
-    "ngettext": true,
-    "interpolate": true,
-    "process": true,
-    "require": true
+  'globals': {
+    'gettext': true,
+    'ngettext': true,
+    'interpolate': true,
+    'process': true,
+    'require': true
   },
-  "env": {
-    "browser": true,
-    "es2021": true
+  'env': {
+    'browser': true,
+    'commonjs': true,
```

Comment:
eslint env: "commonjs - CommonJS global variables and CommonJS scoping (use this for browser-only code that uses Browserify/WebPack)."
https://eslint.org/docs/latest/use/configure/language-options#specifying-environments

Reply:
I am not sure if we need commonjs, I think it is the `var foo = require('bar.js');` syntax which we don't use. (Yay JavaScript.) I will try to find out what dialect we are using.

Reply:
Alright, thanks. This is somehow "Neuland" for me and I just searched for the possible values of env. commonjs (using webpack) sounded promising. But sure, please remove/adapt accordingly.